src/traccuracy/track_errors/_ctc.py

```diff
@@ -111,11 +112,16 @@ def get_edge_errors(gt_graph, comp_graph, node_mapping):
     fn_edges = []
     ws_edges = []
 
+    node_mapping_first = np.array([mp[0] for mp in node_mapping])
+    node_mapping_second = np.array([mp[1] for mp in node_mapping])
+
     # fp edges - edges in induced_graph that aren't in gt_graph
     for edge in tqdm(induced_graph.edges, "Evaluating FP edges"):
         source, target = edge[0], edge[1]
-        source_gt_id = list(filter(lambda mp: mp[1] == source, node_mapping))[0][0]
-        target_gt_id = list(filter(lambda mp: mp[1] == target, node_mapping))[0][0]
+
+        source_gt_id = node_mapping[np.where(node_mapping_second == source)[0][0]][0]
```

Comment:
Just to double check my own understanding - we are still finding the corresponding gt/comp IDs one by one, but it's just that much faster using `np.where` on the prebuilt array?

Reply:
Yes, seems like `filter` is way slower than the numpy intrinsics.